vhdbuilder/packer/install-dependencies.sh

```diff
@@ -413,9 +446,9 @@ done
 # v1.17.11-hotfix.20200901.1
 # v1.17.13
 # v1.18.8-hotfix.20200924
-# v1.18.10
+# v1.18.9
 # v1.19.0
-# v1.19.3
+# v1.19.1-hotfix.20200923
```

Comment:
we need 1.19.3 actually.

Reply:
oops I think I took the wrong changeset when I merged. Reverting.